### PR TITLE
pkg/scaffold: use Role/Binding instead of ClusterRole/Binding

### PR DESCRIPTION
**Pull request by fanminshi**
cc/ @hasbro17

### CODE REVIEW

File: pkg/scaffold/rolebinding.go [hunk 1]
```diff
@@ -54,7 +52,7 @@ subjects:
   name: default
   namespace: default
```
Comment (hasbro17) [inline pkg/scaffold/rolebinding.go]: Remove the ServiceAccount namespace. 
A Role binds to ServiceAccounts in the namespace it's created in.

Comment (fanminshi) [reply pkg/scaffold/rolebinding.go]: cool. Didn't know about that.

Comment (fanminshi) [reply pkg/scaffold/rolebinding.go]: fixed!

**Comment by hasbro17** [comment]
LGTM

**Pull request merged**